modules/functional_expansion_tools/unit/src/CylindricalDuo.C

```diff
@@ -14,9 +14,13 @@
 TEST(FunctionalExpansionsTest, zernikeConstructor)
 {
   const unsigned int order = 5;
+  expansion_type = "orthonormal";
+  generation_type = "standard";
```

Comment:
Do you think it would be a good idea to add a unit that would evaluate the sqrt(mu) normalization as well, or just leave that up to the test from `volume_coupling_custom_norm.i`?

Reply:
Unit tests added!